app/models/nomination_email.rb

```diff
@@ -2,4 +2,14 @@
 
 class NominationEmail < ApplicationRecord
   belongs_to :school
+
+  NOMINATION_EXPIRY_TIME = 7.days
+
+  def nomination_expired?
+    (sent_at + NOMINATION_EXPIRY_TIME) < Time.zone.now
+  end
+
+  def tutor_already_nominated?
```

Comment:
I don't think this belongs in this model. It needs to go in the school class, I believe. We have similar ones there already: https://github.com/DFE-Digital/early-careers-framework/blob/develop/app/models/school.rb#L76

Reply:
done